src/legacy/core_plugins/vis_default_editor/public/components/controls/components/number_list/number_list.tsx

```diff
@@ -77,8 +81,28 @@ function NumberList({
             })
       );
     }
-    setModels(state => getUpdatedModels(numberArray, state, numberRange, id));
-  }, [numberArray, numberRange, validateAscendingOrder]);
+    if (disallowDuplicates) {
+      const duplicateModelIndices = validateUniqueness(numberArray);
+      if (duplicateModelIndices.length) {
+        individualModelErrorMessage = i18n.translate(
+          'visDefaultEditor.controls.numberList.duplicateValueErrorMessage',
+          {
+            defaultMessage: 'The value should not be duplicated.',
```

Comment:
I think this could just say `Duplicated value` unless this copy came from the issue itself.

Reply:
Changed the wording!